torch/_inductor/compile_fx_async.py

```diff
@@ -235,12 +243,15 @@ def __call__(self, args: Sequence[Any]) -> Any:
         return res
 
     def _check_and_switch_progression(self) -> None:
```

Comment:
nit: Might be cleaner to have this and _switch_to_progression_stage members on ProgressiveCompilationState (I think the only self references are self._compilation_state)